invenio_accounts/tasks.py

```diff
@@ -40,3 +45,12 @@ def send_security_email(data):
     msg = Message()
     msg.__dict__.update(data)
     current_app.extensions['mail'].send(msg)
+
+
+@shared_task
+def clean_session_table():
+    """Clean session table."""
+    expired_moment = datetime.now() - current_app.permanent_session_lifetime
+    SessionActivity.query.filter(
```

Comment:
Perhaps we could add it as a method on SessionActivity...something like:

``SessionActivity.query_by_expired().delete()``